extension/jaegerremotesampling/model.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2018 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerremotesampling
+
+// strategy defines a sampling strategy. Type can be "probabilistic" or "ratelimiting"
+// and Param will represent "sampling probability" and "max traces per second" respectively.
+type strategy struct {
```

Comment:
where are these used? Do they need to be in the root package? It seems these define the data types for JSON output, so should be next to http server code.

Reply:
I think http uses the response from the Source interface's GetSamplingStrategy for the JSON output, which is `github.com/jaegertracing/jaeger/proto-gen/api_v2.SamplingStrategyResponse`.

This is used by the filesource to load the strategies